src/DI/TranslationExtension.php

```diff
@@ -121,6 +122,18 @@ public function loadConfiguration(): void
 			$factory = Contributte\Translation\Translator::class;
 		}
 
+		// Add own Translator
+		if ($this->config->translator !== null) {
+			$reflection = new ReflectionClass($this->config->translator);
+			if (!$reflection->isSubclassOf(Contributte\Translation\Translator::class)) {
+				throw new Contributte\Translation\Exceptions\InvalidArgument('Translator must extends class "' . Contributte\Translation\Translator::class . '".');
+			}
+			if (!$reflection->isSubclassOf(Contributte\Translation\DebuggerTranslator::class)) {
```

Comment:
Remove this condition, debugger you can set in configuration

Reply:
Are you sure ? If I set templateFactory to Translator witch extends Contributte\Translation\Translator and in debug mode (localhost) I get TypeError - Argument 1 passed to Contributte\Translation\Tracy\Panel::__construct() must be an instance of Contributte\Translation\DebuggerTranslator

Reply:
You are right, give me some time to make little refactor